src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -311,15 +309,11 @@ bool DirectoryService::VerifyPoWWinner(
                       << ". Will continue look for it in PoW from leader.");
       if (dsWinnerPoWsFromLeader.find(DSPowWinner.first) !=
           dsWinnerPoWsFromLeader.end()) {
-        uint8_t expectedDSDiff = DS_POW_DIFFICULTY;
+        uint8_t expectedDSDiff = m_mediator.m_dsBlockChain.GetLastBlock()
```

Comment:
This change doesn't seem related to the to-do.  Was `expectedDSDiff = DS_POW_DIFFICULTY` a bug?

Reply:
No, previous the genesis block difficulty is not using DS_POW_DIFFICULTY, now it is fixed. So it is kind of code optimization